src/sonic_ax_impl/mibs/vendor/cisco/ciscoPfcExtMIB.py

```diff
@@ -0,0 +1,263 @@
+        return None
+
+    def get_next(self, sub_id):
+        """
+        :param sub_id: The 1-based sub-identifier query.
+        :return: the next sub id.
+        """
+        try:
+            if sub_id is None:
+               return (self.if_range[0][0], 1)
+
+            if len(sub_id) < 2:
+               return (sub_id[0], 1)
+
+            if sub_id[1] >= self.max_prio:
+                return None
+
+            right = sub_id[1] + 1
+
+            return (sub_id[0], right)
+        except Exception as e:
+            mibs.logger.error("failed to get next oid with error = {}".format(str(e)))
+
+    def requestsPerPriority(self, sub_id):
+        """
+        :param sub_id: The 1-based sub-identifier query.
+        :return: the counter for the respective sub_id/table.
+        """        
+        port_oid = ''
+        queue_index = ''
+        try:
+            port_oid = self.get_oid((sub_id[0],))
+            queue_index = self.queue_index(sub_id)
+            if port_oid is None or queue_index is None:
+                raise IndexError()
+        except:
+            mibs.logger.warning("requestsPerPriority: incorrect sub_id = {}".format(str(sub_id)))
+            return None
+
+        counter_name = 'SAI_PORT_STAT_PFC_' + str(queue_index) + '_RX_PKTS'
+
+        if port_oid in self.oid_lag_name_map:
+            counter_value = 0
+            for lag_member in self.lag_name_if_name_map[self.oid_lag_name_map[port_oid]]:
+                counter_value += self._get_counter(mibs.get_index(lag_member), counter_name)
+
+            # truncate to 32-bit counter
+            return counter_value & 0x00000000ffffffff
+        else:
+            return self._get_counter(port_oid, counter_name)
+
+    def indicationsPerPriority(self, sub_id):
+        """
+        :param sub_id: The 1-based sub-identifier query.
+        :return: the counter for the respective sub_id/table.
+        """
+        port_oid = ''
+        queue_index = ''
+        try:
+            port_oid = self.get_oid((sub_id[0],))
+            queue_index = self.queue_index(sub_id)
+            if port_oid is None or queue_index is None:
+                raise IndexError()
+        except:
+            mibs.logger.warning("indicationsPerPriority: incorrect sub_id = {}".format(str(sub_id)))
+            return None
+
+        counter_name = 'SAI_PORT_STAT_PFC_' + str(queue_index) + '_TX_PKTS'
+
+        if port_oid in self.oid_lag_name_map:
+            counter_value = 0
+            for lag_member in self.lag_name_if_name_map[self.oid_lag_name_map[port_oid]]:
+                counter_value += self._get_counter(mibs.get_index(lag_member), counter_name)
+
+            # truncate to 32-bit counter
+            return counter_value & 0x00000000ffffffff
+        else:
+            return self._get_counter(port_oid, counter_name)
+
+
+# cpfcIfTable = '1.1'
+# cpfcIfEntry = '1.1.1.x'
+class cpfcIfTable(metaclass=MIBMeta, prefix='.1.3.6.1.4.1.9.9.813.1.1'):
```

Comment:
>cpfcIfTable [](start = 6, length = 11)

Name it as cpfcIf???Table.

Reply:
Yes, name is "cpfcIfTable" according to DD.

Reply:
"DD" is design document.